catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/main/java/org/codice/alliance/catalog/plugin/defaultsecurity/DefaultSecurityAttributeValuesPlugin.java

```diff
@@ -0,0 +1,207 @@
+                    new MetacardTypeImpl(metacard.getMetacardType()
+                            .getName(),
+                            metacard.getMetacardType(),
+                            SECURITY_ATTRIBUTE_DESCRIPTORS.getAttributeDescriptors()));
+            Set<String> updatedTags = new HashSet<>(metacard.getTags());
+            updatedTags.add(DEFAULTMARKINGS);
+            metacardImpl.setTags(updatedTags);
+            extendedMetacard = metacardImpl;
+        } else {
+            MetacardImpl metacardImpl = new MetacardImpl(metacard);
+            Set<String> updatedTags = new HashSet<>(metacard.getTags());
+            updatedTags.add(DEFAULTMARKINGS);
+            metacardImpl.setTags(updatedTags);
+            extendedMetacard = metacardImpl;
+        }
+
+        Map<String, Attribute> securityMarkings = getHighwaterSecurityMarkings();
+        securityMarkings.keySet()
+                .stream()
+                .filter(securityMarking -> extendedMetacard.getMetacardType()
+                        .getAttributeDescriptor(securityMarking) != null)
+                .forEach(securityMarking -> {
+                    extendedMetacard.setAttribute(securityMarkings.get(securityMarking));
+                });
+        return extendedMetacard;
+    }
+
+    public void setClassification(String classification) {
+        sysHighToMetacardAttributeMapping.put(classification, Security.CLASSIFICATION);
+    }
+
+    public void setReleasability(String releasability) {
+        sysHighToMetacardAttributeMapping.put(releasability, Security.RELEASABILITY);
+    }
+
+    public void setCodewords(String codewords) {
+        sysHighToMetacardAttributeMapping.put(codewords, Security.CODEWORDS);
+    }
+
+    public void setDisseminationControls(String disseminationControls) {
+        sysHighToMetacardAttributeMapping.put(disseminationControls,
+                Security.DISSEMINATION_CONTROLS);
+    }
+
+    public void setOtherDisseminationControls(String otherDisseminationControls) {
+        sysHighToMetacardAttributeMapping.put(otherDisseminationControls,
+                Security.OTHER_DISSEMINATION_CONTROLS);
+    }
+
+    public void setOwnerProducer(String ownerProducer) {
+        sysHighToMetacardAttributeMapping.put(ownerProducer, Security.OWNER_PRODUCER);
+    }
+
+    @Override
+    public CreateRequest process(CreateRequest createRequest)
+            throws PluginExecutionException, StopProcessingException {
+        List<Metacard> updatedMetacards = new ArrayList<>();
+        createRequest.getMetacards()
+                .stream()
+                .filter(Objects::nonNull)
+                .forEach(m -> updatedMetacards.add(addDefaults(m)));
+        return new CreateRequestImpl(updatedMetacards,
+                createRequest.getProperties(),
+                createRequest.getStoreIds());
+    }
+
+    @Override
+    public UpdateRequest process(UpdateRequest updateRequest)
```

Comment:
I'm on the fence about this.  I had that originally but updating existing data without markings would cause it to seemingly disappear. That being said if someone exported all of their metacards into a new instance the new instance would mark some of the data differently so their is also something to be said for consistency.